### PR TITLE
Rename namespaces to use Fishjam

### PR DESCRIPTION
**Pull request by karkakol**
## Description

Change everything in code from jellyfish to fishjam

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/android_emulator_e2e.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 40
     services:
-      jellyfish:
-        image: ghcr.io/fishjam-dev/jellyfish:0.3.0
+      fishjam:
+        image: ghcr.io/fishjam-dev/fishjam:0.6.2
         env:
           JF_CHECK_ORIGIN: false
```
Comment (balins) [inline .github/workflows/android_emulator_e2e.yaml:18]: `FJ` 👀

File: example/android/app/src/main/java/com/jellyfishexample/MainActivity.kt
Comment (balins) [inline example/android/app/src/main/java/com/jellyfishexample/MainActivity.kt:1]: The directory name should be updated as well

**Review by balins — CHANGES_REQUESTED** [review]
Apart from that, you need to update a couple of more files. Search for `JELLYFISH` and `JF`.
Also, looks like the server SDK should be re-generated for `webdriverio-test` (or, even better, just switch to using [`js-server-sdk`](https://github.com/jellyfish-dev/js-server-sdk)).

File: example/ios/Podfile [hunk 2]
```diff
@@ -25,44 +25,14 @@ if linkage != nil
...
-    begin
-      expo_patch_react_imports!(installer)
-    rescue => e
-      Pod::UI.warn e
-    end
-    begin
-      expo_patch_react_imports!(installer)
-    rescue => e
-      Pod::UI.warn e
-    end
```
[the remark this answers is not in the document]
Comment (karkakol) [reply example/ios/Podfile:65]: unnecesary

File: example/ios/Podfile.lock [hunk 3]
```diff
@@ -1,23 +1,23 @@
 PODS:
   - boost (1.83.0)
   - DoubleConversion (1.1.6)
-  - EXConstants (15.4.5):
+  - EXConstants (15.4.6):
```
Comment (mironiasty) [inline example/ios/Podfile.lock:4]: Why these changes in this PR? 🤔

File: package.json [hunk 4]
```diff
@@ -26,24 +26,26 @@
...
   "publishConfig": {
     "registry": "https://registry.npmjs.org/",
     "access": "public"
   },
   "dependencies": {
     "phoenix": "^1.7.6",
     "promise-fs": "^2.1.1",
-    "protobufjs": "^7.2.4"
+    "protobufjs": "^7.2.4",
+    "ts-proto": "^1.176.0"
```
[the remark this answers is not in the document]
Comment (balins) [reply package.json:48]: `ts-proto` is for generating JS code from protobuf definitions. In runtime, we need only `protobufjs`.

File: .github/workflows/android_emulator_e2e.yaml (hunk 1 above)
Comment (karkakol) [reply .github/workflows/android_emulator_e2e.yaml:18]: It was not necessary, but changed.

File: example/.env.example [hunk 5]
```diff
@@ -1,2 +1,2 @@
-JELLYFISH_URL=Your Jellyfish URL
+JELLYFISH_URL=Your Fishjam URL
```
[the remark this answers is not in the document]
Comment (karkakol) [reply example/.env.example:1]: I will change it

File: package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (karkakol) [reply package.json:48]: Yep, it was necessary for generating protos, in past sb removed it from `package.json`

**Comment by karkakol** [comment]
> Apart from that, you need to update a couple of more files. Search for `JELLYFISH` and `JF`. Also, looks like the server SDK should be re-generated for `webdriverio-test` (or, even better, just switch to using [`js-server-sdk`](https://github.com/jellyfish-dev/js-server-sdk)).

I will create seperate ticket for it.

File: package.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (mironiasty) [reply package.json:48]: In that case, should it be added as `devDependency`?

File: example/screens/ConnectWithTokenScreen.tsx [hunk 6]
```diff
@@ -32,20 +32,22 @@ type Props = CompositeScreenProps<
...
   const [connectionError, setConnectionError] = useState<string | null>(null);
 
-  const [peerToken, onChangePeerToken] = useState('');
-  const [jellyfishUrl, onChangeJellyfishUrl] = useState(
-    process.env.JELLYFISH_URL ?? '',
+  const [peerToken, onChangePeerToken] = useState(
+    'SFMyNTY.g2gDdAAAAAJkAAdwZWVyX2lkbQAAACQwNjk1YTk5MC1jNzhmLTQyYWQtYmE4YS1lN2EzYTg5MWIyYjRkAAdyb29tX2lkbQAAAAE3bgYAB44BpY8BYgABUYA.B8k6Z9STfSrcYBZcT_-rPBJJAbHH8Or-zVMU-uPBvpk',
+  );
+  const [fishjamUrl, onChangeFishjamUrl] = useState(
+    process.env.FISHJAM_URL ?? 'ws://192.168.82.189:5002/socket/peer/websocket',
```
Comment (balins) [inline example/screens/ConnectWithTokenScreen.tsx:42]: leftovers

File: package.json [hunk 7]
```diff
@@ -44,7 +44,8 @@
...
```
Comment (balins) [inline package.json:48]: Please install it once again, but as a dev-only dependency

**Review by balins — APPROVED** [review]
Assuming that the whole `webdriverio-test` will be migrated in a separate PR and the two issues below are fixed, I think we're good to go 🚀

File: example/.gitignore [hunk 8]
```diff
@@ -65,5 +65,5 @@ yarn-error.log
 
 # Env
 .env.development
-/ios/JellyfishExample.xcworkspace/contents.xcworkspacedata
+/ios/FishjamExample.xcworkspace/contents.xcworkspacedata
 /ios/jf/
```
Comment (mironiasty) [inline example/.gitignore:69]: Not sure where this line came from, but I think it could be either removed or updated

File: example/webdriverio-test/readme.md [hunk 9]
```diff
@@ -43,7 +43,7 @@
    -e JF_WEBRTC_TURN_IP=[ip address] \
    -e JF_WEBRTC_TURN_LISTEN_IP=0.0.0.0 \
    -e JF_SERVER_API_TOKEN=development \
-   ghcr.io/jellyfish-dev/fishjam:0.3.0
+   ghcr.io/fishjam-dev/fishjam:0.3.0
```
Comment (mironiasty) [inline example/webdriverio-test/readme.md:46]: As you've updated version in `.github/workflows/android_emulator_e2e.yaml`, should it be fixed here too?

Comment (karkakol) [reply example/webdriverio-test/readme.md:46]: yep

File: package.json [hunk 10]
```diff
@@ -43,9 +43,7 @@
   },
   "dependencies": {
     "phoenix": "^1.7.6",
-    "promise-fs": "^2.1.1",
-    "protobufjs": "^7.2.4",
```
Comment (balins) [inline package.json]: We need `protobufjs` here

File: package.json [hunk 11]
```diff
@@ -73,7 +71,9 @@
     "release-it": "^17.0.3",
     "typedoc": "^0.25.7",
     "typedoc-plugin-mark-react-functional-components": "^0.2.2",
-    "typescript": "5.3.x"
+    "typescript": "5.3.x",
+    "protobufjs": "^7.2.4",
```
Comment (balins) [inline package.json:75]: We don't need `protobufjs` here

**Review by balins — CHANGES_REQUESTED** [review]
Please fix the dependencies and make sure the `package-lock.json` gets updated accordingly.

**Review by balins — APPROVED** [review]
LGTM. Just remember to create a ticket for `webdriverio-test` migration.

**Comment by balins** [comment]
I've overlooked this, but I think we should rename to react-native-client before releasing. We should be consistent with ts-client and react-client. You could do this here or in a separate PR.
